src/date.js

```diff
@@ -63,3 +63,15 @@ export function withinDates (min, max, date) {
   const mDate = moment(date)
   return mDate.isBetween(min, max, 'day') || mDate.isSame(min, 'day') || mDate.isSame(max, 'day')
 }
+
+export function setDateToZeroTime(date) {
+  date.setHours(0)
+  date.setMinutes(0)
+  date.setSeconds(0)
```

Comment:
You are modifying the passed object which may come back and haunt you in your sleep. Care to solve this in immutable way?